client/src/components/screens/PatientVariant/index.jsx

```diff
@@ -46,82 +50,73 @@ class PatientVariantScreen extends React.Component {
     }
   }
 
-  componentDidMount() {
-    //@NOTE PA00002 currently is the only patient with indexed data.
-    this.props.actions.searchVariants('PA00002', [{key:'aggs', instructions:[]}], 'aggs', 'impact', 0, 1);
+  shouldComponentUpdate(nextProps, nextState) {
```

Comment:
I think this falls outside the scope of the ticket and is a good example of eager optimization.

Reply:
See my comments:
https://github.com/cr-ste-justine/clin-frontend/pull/91/files#r340380669
https://github.com/cr-ste-justine/clin-frontend/pull/91/files#r340381061